dt-posts/posts.php

```diff
@@ -286,11 +286,11 @@ public static function format_connection_message( $p2p_id, $activity, $action =
             }
         } else if ( $p2p_type === "contacts_to_peoplegroups" || $p2p_type === "groups_to_peoplegroups"){
             if ($action == "connected to"){
-                $object_note_to = sprintf( esc_html_x( '%1$s added as people group on %2$s', 'Deaf added as people group on contact1', 'disciple_tools' ), $to_title, $from_title );
-                $object_note_from = sprintf( esc_html_x( '%s added to people groups', 'Deaf added to people groups', 'disciple_tools' ), $to_title );
+                $object_note_to = sprintf( esc_html_x( '%1$s added as people group on %2$s', 'Shaikh added as people group on contact1', 'disciple_tools' ), $to_title, $from_title );
```

Comment:
I changed this context because the word Deaf - which is as I found out the name of a people group - is also the word for someone who can't hear. This new people group name of Shaikh should be less confusing.